src/sentry/static/sentry/app/views/performance/transactionDetails/styles.tsx

```diff
@@ -10,13 +11,21 @@ type MetaDataProps = {
   tooltipText: string;
   bodyText: React.ReactNode;
   subtext: React.ReactNode;
+  beta?: boolean;
 };
 
-export function MetaData({headingText, tooltipText, bodyText, subtext}: MetaDataProps) {
+export function MetaData({
+  headingText,
+  tooltipText,
+  bodyText,
+  subtext,
+  beta,
+}: MetaDataProps) {
   return (
     <HeaderInfo>
       <StyledSectionHeading>
         {headingText}
+        {beta && <StyledFeatureBadge type="beta" />}
```

Comment:
Can we move the beta badge after the question tooltip?

Reply:
![image](https://user-images.githubusercontent.com/4205004/113066722-0546b880-9189-11eb-825e-3e4bb081efa5.png)